package.json

```diff
@@ -12,7 +12,25 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "@nx/js": "19.1.2",
+    "@nx/react": "19.1.2",
+    "@nx/storybook": "19.1.2",
+    "@nx/web": "19.1.2",
+    "@storybook/addon-essentials": "7.6.20",
+    "@storybook/addon-interactions": "^7.5.3",
+    "@storybook/core-server": "7.6.20",
+    "@storybook/jest": "^0.2.3",
+    "@storybook/react-vite": "7.6.20",
+    "@storybook/test-runner": "^0.13.0",
+    "@storybook/testing-library": "^0.2.2",
+    "@swc-node/register": "~1.9.1",
+    "@swc/core": "~1.5.7",
+    "@swc/helpers": "~0.5.11",
     "nx": "^19.1.2",
+    "prettier": "^2.6.2",
```

Comment:
Not sure how I ended up adding prettier but I think we do want it here right? bloom-player has it in devDependencies also